### PR TITLE
Fix typos in README file

### PR DESCRIPTION
**Pull request by bolatovumar**
This PR fixes a few types in the README file.

### CODE REVIEW

File: BTCPayServer.Tests/README.md [hunk 1]
```diff
@@ -88,8 +88,8 @@ Or, uncheck the box that says, "Break when this exception type is thrown".
 
 ### `docker-compose up dev` failed or tests are not passing, what should I do?
 
-1. Run `docker-compose down --v` (this will reset your test environment)
```
Comment (bolatovumar) [inline BTCPayServer.Tests/README.md:91]: The `--v` parameter doesn't exist and you will get an error if you try to use it.

Comment (pavlenex) [reply BTCPayServer.Tests/README.md:91]: Are you sure I use `docker-compose down --v` all the time?

Comment (dennisreimann) [reply BTCPayServer.Tests/README.md:91]: I think they recently deprecated it, we had to update in to use `--volumes` in the [Lightning lib as well](https://github.com/btcpayserver/BTCPayServer.Lightning/commit/9fd83188429134c1f05a5f3112280276550956c0).

File: BTCPayServer.Tests/README.md [hunk 2]
```diff
@@ -88,8 +88,8 @@ Or, uncheck the box that says, "Break when this exception type is thrown".
...
-2. Run `docker-compose pull` (this will ensure you have the lastest images)
+1. Run `docker-compose down` (this will reset your test environment)
```
Comment (dennisreimann) [inline BTCPayServer.Tests/README.md:91]:
```suggestion
1. Run `docker-compose down --volumes` (this will reset your test environment)
```

**Review by dennisreimann — APPROVED** [review]

**Pull request merged**